ui/src/common/useAlbumsPerPage.js

```diff
@@ -18,10 +15,5 @@ const getPerPageOptions = (width) => {
 }
 
 export const useAlbumsPerPage = (width) => {
-  const perPage =
-    useSelector((state) =>
-      get(state.admin.resources, ['album', 'list', 'params', 'perPage'])
```

Comment:
This call was in place to make sure that when you navigate to a Album List ("Random", "Recently Playerd", ...) the selected perPage value is kept. Without this, it always reset to the default (15 for list, 18 for grid). What was the reasoning to remove this?

Reply:
If we adjust it to only use `useAlbumsPerPage()` on the grid view, when the user switches from list to grid, it will incorrectly pull the perPage setting that was stored in the state for list and then have the same issue of not matching the perPage options presented. It does have the side-effect you described. I fixed this with some refactoring in the pushed commit. I'm not sure if that's the best way to handle the URL query string.

It may make sense to step back a level and figure out if we actually want to use different pagination options for grid/list view, and if not, maybe we just adjust the song/artist lists to use the same dynamic per-page options as we do for album lists. My initial inspiration for this was finding it odd that my options for Song/Album lists were `15, 25, 50`, but for Album lists `12, 24, 48`. The intent of the PR was to bring those inline, while still keeping the grid view dynamic based on breakpoint.